exercises/collatz-conjecture/example.rs

```diff
@@ -0,0 +1,13 @@
+// return Ok(x) where x is the number of steps required to reach 1
+pub fn collatz(n: u64) -> Result<u64, &'static str> {
+    if n == 0 {
+        return Err("Only positive numbers are allowed");
```

Comment:
For the purposes of consistency with the `else` case, the `return` and semicolon should be removed here.